blog/2023-ecosystem-eoy-recap.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Electron's Ecosystem 2023 Recap
+date: 2023-11-30T00:00:00.000Z
+authors:
+  - name: erickzhao
+    url: 'https://github.com/erickzhao'
+    image_url: 'https://github.com/erickzhao.png?size=96'
+slug: ecosystem-2023-eoy-recap
+---
+
+Reflecting on the improvements and changes in Electron's developer ecosystem in 2023.
+
+---
+
+In the past few months, we've been cooking up some changes across the Electron ecosystem to supercharge the developer experience for Electron apps! Here’s a swift rundown of the latest additions straight from Electron HQ.
+
+## Electron Forge 7 and beyond
+
+Electron Forge 7 — the newest major version of our all-in-one tool for packaging and distributing Electron applications — is now available.
+
+While Forge 6 was a complete rewrite from v5, v7 is smaller in scope but still contains a few breaking changes. Going forward, we will continue to publish major versions of Forge as breaking changes need to be made.
+
+For more details, see the full [Forge v7.0.0 changelog](https://github.com/electron/forge/releases/tag/v7.0.0) on GitHub.
+
+### Breaking changes
+
+- **Switched to `notarytool` for macOS notarization:** As of 2023-11-01, Apple sunset the legacy `altool` for macOS notarization, and this release removes it from Electron Forge entirely.
+- **Minimum Node.js increased to v16.0.0:** With this release, we’ve set the minimum required Node.js version to 16.0.0.
+- **Dropped support for `electron-prebuilt` and `electron-prebuilt-compile`**: `electron-prebuilt` [was the original name for Electron’s npm module](https://www.electronjs.org/blog/npm-install-electron), but was replaced by `electron` in v1.3.1. `electron-prebuilt-compile` was an alternative to that binary that came with enhanced DX features, but was eventually abandoned as a project.
+
+### Highlights
+
+- **Google Cloud Storage publisher:** As part of our push to better support static auto updating, Electron Forge now supports publishing directly to Google Cloud Storage!
+- **ESM forge.config.js support:** Electron Forge now supports ESM `forge.config.js` files. (P.S. Look forward to ESM entrypoint support in Electron 28.)
+- **Makers now run in parallel:** In Electron Forge 6, Makers ran sequentially for ✨ legacy ✨ reasons. Since then, we’ve tested out parallelization for the Make step with no adverse side effects, so you should see a speed-up when building multiple targets for the same platform!
```

Comment:
Add PR link

```suggestion
- **[Google Cloud Storage publisher](https://github.com/electron/forge/pull/2100):** As part of our push to better support static auto updating, Electron Forge now supports publishing directly to Google Cloud Storage!
- **[ESM forge.config.js support](https://github.com/electron/forge/pull/3358):** Electron Forge now supports ESM `forge.config.js` files. (P.S. Look forward to ESM entrypoint support in Electron 28.)
- **[Makers now run in parallel](https://github.com/electron/forge/pull/3363):** In Electron Forge 6, Makers ran sequentially for ✨ legacy ✨ reasons. Since then, we’ve tested out parallelization for the Make step with no adverse side effects, so you should see a speed-up when building multiple targets for the same platform!
```